game_eggs/steamcmd_servers/night_of_the_dead/Night of the dead.json

```diff
@@ -0,0 +1,31 @@
+    ],
+    "file_denylist": [],
+    "startup": "cp \/home\/container\/notd\/ServerSettings.ini \/home\/container\/notd\/LF\/Saved\/Config\/ServerSettings.ini;\r\ncp \/home\/container\/notd\/steamclient64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/steamclient64.dll;\r\ncp \/home\/container\/notd\/tier0_s64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/tier0_s64.dll;\r\ncp \/home\/container\/notd\/vstdlib_s64.dll \/home\/container\/notd\/LF\/Binaries\/Win64\/tier0_s64.dll;\r\ncd \/home\/container\/notd\/; \r\nwine \/home\/container\/notd\/LF\/Binaries\/Win64\/LFServer.exe ?listen -log -Port=7777 -QueryPort=27015 -CRASHREPORTS",
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"LOAD COMPLETE\",\r\n    \"userInteraction\": []\r\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# Night of the dead Installation Script\r\n#\r\n## Define variables\r\nexport HOME=\/mnt\/server\r\n\r\n## Install dependencies\r\ndpkg --add-architecture i386\r\napt update\r\napt -y --no-install-recommends install curl unzip libstdc++6 lib32gcc1 ca-certificates libsdl2-2.0-0:i386 cabextract\r\napt install -y --install-recommends wine wine64 xvfb\r\n\r\n## download and install steamcmd\r\ncd \/tmp\r\nmkdir -p \/mnt\/server\/steamcmd\r\ncurl -sSL -o steamcmd.tar.gz https:\/\/steamcdn-a.akamaihd.net\/client\/installer\/steamcmd_linux.tar.gz\r\ntar -xzvf steamcmd.tar.gz -C \/mnt\/server\/steamcmd\r\ncd \/mnt\/server\/steamcmd\r\n\r\nchown -R root:root \/mnt\r\n\r\n## Install game using steamcmd\r\nmkdir $HOME\/notd\r\n.\/steamcmd.sh +login anonymous +@sSteamCmdForcePlatformType windows +force_install_dir $HOME\/notd +app_update 1420710 validate +quit\r\n\r\n## making folder for config file\r\nmkdir $HOME\/notd\/LF\/Saved\r\nmkdir $HOME\/notd\/LF\/Saved\/Config\r\n\r\n## set up 32 bit libraries\r\nmkdir -p $HOME\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so\r\n\r\n## set up 64 bit libraries\r\nmkdir -p $HOME\/.steam\/sdk64\r\ncp -v linux64\/steamclient.so ..\/.steam\/sdk64\/steamclient.so\r\n\r\n## Install WineTricks\r\nexport DISPLAY=:0.0\r\nexport WINEPREFIX=$HOME\/.wine\r\nrm -r $HOME\/.wine\r\nrm -r $HOME\/.cache\r\nmkdir -p $HOME\/.wine\r\nmkdir \/tmp\/winetricks\r\ncurl https:\/\/raw.githubusercontent.com\/Winetricks\/winetricks\/master\/src\/winetricks -o \/tmp\/winetricks\/winetricks\r\nchmod +x \/tmp\/winetricks\/winetricks\r\nXvfb :0 -screen 0 1024x768x16 &\r\nenv WINEDLLOVERRIDES=\"mscoree=d;mshtml=d\" wineboot --init --force \/nogui\r\nenv \/tmp\/winetricks\/winetricks -q vcrun2017  \r\nenv \/tmp\/winetricks\/winetricks -q vcrun2013 \r\nenv \/tmp\/winetricks\/winetricks -q sound=disabled  \r\nenv \/tmp\/winetricks\/winetricks -q corefonts\r\nenv \/tmp\/winetricks\/winetricks -q --force dotnet48",
+            "container": "debian:buster-slim",
```

Comment:
You won't need to use wine in the Insall Script. This was a solution that was specifically needed for Space Engineer. But unfortunately, this also leads to an installation that takes an eternity. 

So remove 
```
apt install -y --install-recommends wine wine64 xvfb
```
and
```
## Install WineTricks
export DISPLAY=:0.0
export WINEPREFIX=$HOME/.wine
rm -r $HOME/.wine
rm -r $HOME/.cache
mkdir -p $HOME/.wine
mkdir /tmp/winetricks
curl https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks -o /tmp/winetricks/winetricks
chmod +x /tmp/winetricks/winetricks
Xvfb :0 -screen 0 1024x768x16 &
env WINEDLLOVERRIDES="mscoree=d;mshtml=d" wineboot --init --force /nogui
env /tmp/winetricks/winetricks -q vcrun2017  
env /tmp/winetricks/winetricks -q vcrun2013 
env /tmp/winetricks/winetricks -q sound=disabled  
env /tmp/winetricks/winetricks -q corefonts
env /tmp/winetricks/winetricks -q --force dotnet48
```
from the script. Everything will be faster and it will still work.

Reply:
I also removed wget.